index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="./style.css">
+</head>
+<body>
+    <!-- Header section with h1 element-->
```

Comment:
- [ ] You do have commented out code in your project. Refer to the first feedback I left which addressed this.
- [ ] It would be better if you wrapped your `h1` tag around a `header`. This would make you used Semantics HTML which is a highly recommended industry best practice.